src/java/com/puppetlabs/http/client/impl/PersistentAsyncHttpClient.java

```diff
@@ -15,13 +15,16 @@
     private CloseableHttpAsyncClient client;
     private MetricRegistry metricRegistry;
     private String metricNamespace;
+    private Boolean useURLMetrics;
 
     public PersistentAsyncHttpClient(CloseableHttpAsyncClient client,
                                      MetricRegistry metricRegistry,
-                                     String metricNamespace) {
+                                     String metricNamespace,
+                                     Boolean useURLMetrics) {
```

Comment:
I think you should always use `Boolean` and have the full package be listed as an `import` but since its `java.lang` I guess you don't need to do that?